applications/ParticleMechanicsApplication/custom_utilities/mpm_search_element_utility.h

```diff
@@ -166,7 +355,7 @@ namespace MPMSearchElementUtility
                     bool is_found = SearchStructure.FindPointOnMesh(xg[0], N, pelem, result_begin, MaxNumberOfResults, Tolerance);
 
                     if (is_found == true) {
-                        pelem->Set(ACTIVE);
+                        //pelem->Set(ACTIVE);
```

Comment:
please do remove delicate commented code like this (if it is intended!)

Reply:
dunzo 🚀